about.html

```diff
@@ -15,11 +15,11 @@
 <header>
   <img class="logo" src="img/lambda-black.png">
     <nav>
-      <a href="file:///C:/Users/ijerr/Documents/GitHub/Sprint-Challenge--User-Interface/index.html#">Home</a>
-      <a href="file:///C:/Users/ijerr/Documents/GitHub/Sprint-Challenge--User-Interface/about.html">About</a>
-      <a href="#">Products</a>
-      <a href="#">Blog</a>
-      <a href="#">Contact</a>
+      <a href="./index.html">Home</a>
+      <a href="./About.html">About</a>
+      <a href="./Product.html">Products</a>
```

Comment:
good job with remembering the links. dont forget to add the a tag around the image so its clickable